librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -3,6 +3,251 @@
 ---
 name: cmd_debug
 commands:
+  - name: db
+    summary: Breakpoints commands
+    subcommands:
+      - name: db
+        summary: List breakpoints / Add breakpoint
+        cname: cmd_debug_db
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
+            optional: true
+      - name: db*
+        summary: List breakpoints in r commands
+        cname: cmd_debug_list_bp_r
+        args: []
+      - name: dbH
+        summary: Add hardware breakpoint
+        cname: cmd_debug_add_hw_bp
+        args:
+          - name: addr
+            type: RZ_CMD_ARG_TYPE_RZNUM
```

Comment:
let's remove all these `addr` args and just assume that the command refers to `core->offset`, so users have to use `dbH @ 0x112233` to set a bp somewhere.

Reply:
Remove them for all commands?

Reply:
Yes, at least where there are the sole arguments, also the last argument.